ml-agents-envs/mlagents_envs/env_utils.py

```diff
@@ -97,6 +97,7 @@ def launch_executable(file_name: str, args: List[str]) -> subprocess.Popen:
                 # This is generally good since we want the environment to have a chance to shutdown,
                 # but may be undesirable in come cases; if so, we'll add a command-line toggle.
                 # Note that on Windows, the CTRL_C signal will still be sent.
+                stdout=subprocess.DEVNULL,
```

Comment:
The above comment goes with `start_new_session` - please move the new line above the comment or below `start_new_session`